packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-constraint-group-form.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+	<div class="container">
+		<div class="form-header">
+			<div>
+				<InputText
+					v-if="isEditing"
+					v-model="config.name"
+					placeholder="Constraint name"
+					@focusout="emit('update-self', config)"
+				/>
+				<h6 v-else>{{ props.constraint.name }}</h6>
+				<i
+					:class="{ 'pi pi-check i': isEditing, 'pi pi-pencil i': !isEditing }"
+					:style="'cursor: pointer'"
+					@click="onEdit"
+				/>
+			</div>
+			<label for="active">Active</label>
+			<InputSwitch v-model="config.isActive" @change="emit('update-self', config)" />
+			<i class="trash-button pi pi-trash" @click="emit('delete-self')" />
+		</div>
+		<div class="section-row">
+			<p>The average value of</p>
+			<Dropdown
+				class="p-inputtext-sm"
+				:options="modelStateAndObsOptions"
+				v-model="config.targetVariable"
+				placeholder="Select"
+				@update:model-value="emit('update-self', config)"
+			/>
+			<p>at</p>
+			<Dropdown
+				class="p-inputtext-sm"
+				:options="[
+					{ label: 'Max', value: ContextMethods.max },
+					{ label: 'Day average', value: ContextMethods.day_average }
+				]"
+				option-label="label"
+				option-value="value"
+				v-model="config.qoiMethod"
+				@update:model-value="emit('update-self', config)"
+			/>
+			<p>over the worst</p>
+			<InputNumber
+				class="p-inputtext-sm"
+				inputId="integeronly"
+				v-model="config.riskTolerance"
+				@focusout="emit('update-self', config)"
+			/>
+			<p>% of simulated outcomes</p>
+			<Dropdown
+				class="toolbar-button"
+				v-model="config.isMinimized"
+				optionLabel="label"
+				optionValue="value"
+				:options="[
+					{ label: 'less than', value: true },
+					{ label: 'greater than', value: false }
+				]"
+				@update:model-value="emit('update-self', config)"
+			/>
+			<p>a threshold of</p>
+			<InputNumber
```

Comment:
Do we want to use this or the `tera-input-number.vue` @blanchco made?

Reply:
Will use tera-input
Thanks